package.json

```diff
@@ -140,6 +140,7 @@
     "full-icu": "^1.3.0",
     "identity-obj-proxy": "^3.0.0",
     "ignore-styles": "^5.0.1",
+    "is-mingw": "^2.0.0",
```

Comment:
i don't need this.

i can do this instead:
`const isMinGW = process.env.MSYSTEM === 'MINGW64' || process.env.MSYSTEM === 'MINGW32';`